Formula/defaultbrowser.rb

```diff
@@ -0,0 +1,30 @@
+class Defaultbrowser < Formula
+  desc "Command-line tool for getting & setting the default browser"
+  homepage "https://github.com/kerma/defaultbrowser"
+  url "https://github.com/kerma/defaultbrowser/archive/v1.0.tar.gz"
+  sha256 "c36cc05e778233f7a3c6109ca78227f82dbdde33a5194dae1457cd83fdf27df2"
+  head "https://github.com/kerma/defaultbrowser.git"
+
+  def install
+    if build.head?
```

Comment:
If the build systems for release and head are entirely different, we'll either avoid having head or wait for a new version to be released. This is essentially two formulas in one :)

Reply:
@fxcoudert 
> This is essentially two formulas in one :)

Well, yes, it is. The head version of the program is quite different and not just with the build system. I would prefer not to have to wait for the new version to be released just so that I can use the new features of the program. I already have some scripts which are dependent on the new behavior and removing the code for the head version would mean that I have to maintain a local clone of the repository anyway, which basically defeats the point of having made this PR.

I mean, we're only talking about two `if` statements being removed. The extra logic encompasses about ~10 lines. It's not that bad, is it?